Rubberduck.Parsing/VBA/RubberduckParserState.cs

```diff
@@ -6,6 +6,7 @@
 using System.Linq;
 using System.Runtime.InteropServices;
 using System.Threading;
+using Antlr4.Runtime;
```

Comment:
huh? Where does that come from?

Reply:
Not for this PR but a prior merge apparently lost that change; the `next` now can't build without that `using`.